css/src/atomics/index.scss

```diff
@@ -1,5 +1,6 @@
 @import './typography.scss';
 @import './border.scss';
+@import './background-pattern.scss';
```

Comment:
shoot, file should be background-image?

Reply:
don't feel strongly about it though

Reply:
New PR for comment is here - https://github.com/microsoft/atlas-design/pull/427 :) Closing the current one